faq

```diff
@@ -0,0 +1 @@
+please tell me how did you add folders in github. 
```

Comment:
https://stackoverflow.com/questions/18773598/creating-folders-inside-github-com-repo-without-using-git

Reply:
Ya I got it.thanks bdy

Reply:
Have fun with git